pmlc/conversion/tile_to_pxa/tests/reshape.mlir

```diff
@@ -1,44 +1,44 @@
 // RUN: pmlc-opt -convert-tile-to-pxa -canonicalize -cse %s | FileCheck %s
 
 module {
-  func @reshaper(%arg0: tensor<1x1x60xf32>) -> tensor<60xf32> {
+  func @reshaper0(%arg0: tensor<1x1x60xf32>) -> tensor<60xf32> {
     %c60 = tile.constant 60
     %0 = "tile.reshape"(%arg0, %c60) : (tensor<1x1x60xf32>, index) -> tensor<60xf32>
```

Comment:
We added a custom printer/parser (via the declarative assembly) but this is using the generic format.

Reply:
Oops, nevermind :)